### PR TITLE
CDP-346 Connect portal to metamask mobile

### PR DESCRIPTION
**Pull request by sirromdev**

### CODE REVIEW

File: src/utils/dev.js [hunk 1]
```diff
@@ -11,3 +11,14 @@ export const uniqueId = (() => {
     return map.get(object);
   };
 })();
+
+export const isMobile = () => {
+  if (sessionStorage.desktop) return false;
```
Comment (levity) [inline src/utils/dev.js]: these properties are new to me -- if you're getting them from some online documentation that explains how they're set, and across which browsers, please add a link to that documentation in a comment here

Comment (levity) [reply src/utils/dev.js]: i created `utils/dev` for functions that would only be used in development--if something is going to be part of the production code, it should probably go elsewhere. perhaps in a new `utils/browser` (IF it still needs to be used--see below)

File: src/utils/ethereum.js [hunk 2]
```diff
@@ -64,9 +65,16 @@ export async function checkEthereumProvider() {
   return new Promise(async (res, rej) => {
     if (typeof window.ethereum !== 'undefined') {
       const accounts = await window.ethereum.enable();
-
       // Keeping selectedAddress for backwards-compatibility.
-      const { selectedAddress, networkVersion } = window.ethereum;
+      let selectedAddress, networkVersion;
```
Comment (levity) [inline src/utils/ethereum.js]: how about this:
```js
const selectedAddress = window.ethereum._selectedAddress || window.ethereum.selectedAddress;
const networkVersion = window.ethereum._networkVersion || window.ethereum.networkVersion;

File: src/utils/dev.js (hunk 1 above)
Comment (sirromdev) [reply src/utils/dev.js]: That function was put together as a blend of [this](https://stackoverflow.com/a/16080250) and [this](https://stackoverflow.com/a/49425890). 

I have removed this in favour of the approach below

**Pull request merged**